runtime/lua/vim/ui.lua

```diff
@@ -0,0 +1,32 @@
+local M = {}
+
+--- Prompts the user to pick a single item from a collection of entries
+---
+---@param entries table Arbitrary entries
```

Comment:
Maybe `items` and `items` would fit better? (and `format_item` instead of `format_entry`)

I don't really have an opinion on it.